test/deploy-time-budget.sh

```
@@ -0,0 +1,148 @@
+deploy_exit=$?
+elapsed=$SECONDS
+
+if [ "$deploy_exit" -ne 0 ]; then
+    echo "FAIL: dbdeployer deploy replication failed (exit=$deploy_exit)"
+    exit 1
+fi
+
+echo "# Deploy completed in ${elapsed}s"
+
+# Verify the replica responds immediately (no race, issue #131)
+SB_DIR=$(ls -d ~/sandboxes/rsandbox_* 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)